external/source/exploits/CVE-2019-0808/NtUserMNDragOverExploit/dllmain.cpp

```diff
@@ -0,0 +1,668 @@

```

Comment:
Umm so this is a bit of an issue as right now this being allocated here and then we are copying the attacker provided shellcode into a limited 4105 buffer; if we use payloads larger than 4105 bytes there is currently no restrictions preventing this from being a buffer overflow.

Reply:
Confirmed resolved, thanks for fixing this @timwr, new code looks much better :)